apps/testrunner/actions/phantomas_test.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+import phantomas
+
+from . import Action
+from phantomas.results import Runs
+
+
+logger = logging.getLogger(__name__)
+
+
+class PhantomasTest(Action):
+    REQUIRED_PARAMS = (
+        'url',
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(PhantomasTest, self).__init__(*args, **kwargs)
+
+    def run(self):
+        try:
+            phantomas_runner = phantomas.Phantomas(
+                url=self.params['url'],
+                runs=self.params['retries']
+            )
+            phantomas_result = phantomas_runner.run()
+            if hasattr(phantomas_result,'_runs'):
```

Comment:
Well how about this:
```python
for metric in result.get_metrics():
    offender = result.get_offenders(metric)
```
or use list comprehension to get all offenders for all metrics
I would stick to a public API which is somewhat guaranteed not to change (http://en.wikipedia.org/wiki/Law_of_Demeter).